src/object/srv_ec_aggregate.c

```diff
@@ -2162,6 +1941,89 @@ agg_data_extent(struct ec_agg_param *agg_param, vos_iter_entry_t *entry,
 	return rc;
 }
 
+/* Iterator call back sub-function for handling data extents.
+ */
+static int
+agg_data_extent(struct ec_agg_param *agg_param, vos_iter_entry_t *entry,
+		struct ec_agg_entry *agg_entry, unsigned int *acts)
+{
+	daos_recx_t	new_recx = entry->ie_recx;
+	daos_off_t	cur_stripenum;
+	daos_off_t	this_stripenum;
+	daos_off_t	this_end_stripenum;
+	int		rc;
+
+	D_ASSERT(!(entry->ie_recx.rx_idx & PARITY_INDICATOR));
+	if (entry->ie_minor_epc == VOS_SUB_OP_MAX)
```

Comment:
This is a valid data extent.  Removed extents are passed at VOS_SUB_OP_MAX+1.  Furthermore, VOS_SUB_OP_MAX is only used when it's not a distributed transaction.  So if you're needing to skip these, something else is wrong.